UnityProject/Assets/Plugins/Android/MainApp.androidlib/src/main/AndroidManifest.xml

```diff
@@ -30,6 +30,7 @@
             android:name=".MainUnityGameActivity"
             android:configChanges="mcc|mnc|locale|touchscreen|keyboard|keyboardHidden|navigation|orientation|screenLayout|uiMode|screenSize|smallestScreenSize|fontScale|layoutDirection|density"
             android:hardwareAccelerated="false"
+            android:theme="@style/AppTheme.NoActionBar"
```

Comment:
I think it would be better to match Unity themes here. See https://github.cds.internal.unity3d.com/unity/unity/blob/trunk/PlatformDependent/AndroidPlayer/Resources/UnityManifest.xml

Could you set
```
android:theme="@style/UnityThemeSelector"
```
for Activity

and 
```
android:theme="@style/BaseUnityGameActivityTheme"
```
for GameActivity?

Reply:
That makes sense! 
Done